core/src/main/java/com/netease/arctic/utils/CatalogUtil.java

```diff
@@ -185,12 +186,25 @@ public static boolean isIcebergCatalog(ArcticCatalog arcticCatalog) {
    */
   public static Map<String, String> mergeCatalogPropertiesToTable(Map<String, String> tableProperties,
                                                                   Map<String, String> catalogProperties) {
-    Map<String, String> mergedProperties = new HashMap<>();
-    catalogProperties.forEach((key, value) -> {
-      if (key.startsWith(CatalogMetaProperties.TABLE_PROPERTIES_PREFIX)) {
-        mergedProperties.put(key.substring(CatalogMetaProperties.TABLE_PROPERTIES_PREFIX.length()), value);
-      }
-    });
+    Map<String, String> mergedProperties = catalogProperties.entrySet().stream()
+            .filter(e -> e.getKey().startsWith(CatalogMetaProperties.TABLE_PROPERTIES_PREFIX))
+            .collect(Collectors.toMap(e ->
+                    e.getKey().substring(CatalogMetaProperties.TABLE_PROPERTIES_PREFIX.length()), Map.Entry::getValue));
+
+    if (!PropertyUtil.propertyAsBoolean(tableProperties, TableProperties.ENABLE_LOG_STORE,
+            TableProperties.ENABLE_LOG_STORE_DEFAULT)) {
+      mergedProperties = mergedProperties.entrySet().stream()
+              .filter(e -> !e.getKey().startsWith(CatalogMetaProperties.LOG_STORE_PROPERTIES_PREFIX) &&
+                      !e.getKey().equals("log.consistency.guarantee.enable"))
```

Comment:
Can we rename the property name `log.consistency.guarantee.enable` to `log-store.consistency-guarantee.enable`?  @YesOrNo828  @zstraw 
We can do this in another PR

Reply:
#745 complete this.